modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectAction/useObjectActionForm.ts

```diff
@@ -112,7 +113,12 @@ export function useObjectActionForm({
 
 		if (Object.keys(errors).length) {
 			openToast({
-				message: REQUIRED_MSG,
+				message: sub(
+					Liferay.Language.get(
+						'only-x-characters-are-allowed-in-the-x-field'
+					),
+					['41', 'name']
+				),
```

Comment:
If you add this message in this condition, when the user keep the ObjectAction name empty, the message on toast will be: `Only 41 characters are allowed in the name field`. I believe this message isn't idea for the empty field. 

So, I believe that you need a new conditional for length > 41, same like this:
```javascript
if (Object.keys(errors).length) {
	openToast({
		message: sub(
			Liferay.Language.get(
				'only-x-characters-are-allowed-in-the-x-field'
			),
			['41', 'name']
		),
		type: 'danger',
	});
}
```

What do you thing?

Reply:
Hi @murilostodolni, it makes sense, sorry for missing that. I will work on that now!

Reply:
@murilostodolni, I was about to commit my changes but I went to test the previous behavior in master to check if my changes weren't breaking anything and I saw that this message is already being brought from another file, probably the ones I've already altered, so there's no need to alter this one. I'll just remove my changes and update it now.